docs/1.concepts/abstraction/chain-signatures.md

```diff
@@ -4,22 +4,23 @@ title: Chain Signatures
 sidebar_label: What are Chain Signatures?
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+Chain Signatures unlock the ability for a single account to transact across multiple blockchain protocols, giving ownership of cross-chain accounts, data, and assets to one NEAR account.
 
-Chain Signatures unlock the ability for a single account to transact across multiple blockchain protocols, giving ownership of many accounts, data, and assets to one NEAR account. 🤯
+This many-to-one ownership is made possible through a mixture of services across our tech stack:
 
-This many-to-one ownership is made possible through NEAR's [unique account model](../basics/accounts/introduction.md), which allows many keys to be associated with a single account. Chain Signatures generates and uses one of these keys to sign payloads compatible with other blockchain platforms but still linked to a native NEAR account.
```

Comment:
@gagdiez, why remove this section? I feel it's essential to link the account abstraction piece here and how the ability for one account to possess many keys makes this whole approach possible. Eth devs are used to a 1 key 1 account and helping them understand a new mental model is crucial for getting this concept.

Reply:
I thought that the key is derived from `accountId` and `path`, and the account keys are not involved in that

Reply:
Correct, but how the new key is created wasn't what I was trying to convey to the reader. I was trying to articulate that "All this is possible (many cross-chain accounts controlled by 1 NEAR account) because of our access keys & account model (that allows for many keys for one account).

Perhaps the second sentence should be:

`Chain Signatures utilizes this ability to generate new keys associated with the NEAR account and sign payloads compatible with other blockchain platforms.`

I think a visual would be helpful here, happy to work on one :)